panels/datetime/cc-datetime-panel.c

```diff
@@ -359,8 +369,12 @@ update_time (CcDateTimePanel *self)
   /* Update the time on the listbow row */
   if (use_ampm)
     {
+      gchar *format_string = g_strdup_printf ("%s %s", _("%e %B %Y, %l:%M"),
```

Comment:
I tested this and I'd swear that the new resultant strings got translated as well, but I see what you mean: just adding the AM/PM string when fallback is true would work, since the translation for %p should be an empty string, so better use that. Will change that.